### PR TITLE
Javascript best practices.

### PR DESCRIPTION
**Pull request by Fene-87**
In this pull request I would like to request for a review that would help me check if I have followed the Javascript best practices.
-Check if the code follows the DRY, KISS and Yagni rules.
-Check if HTML and CSS best practices have been followed.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import _ from 'lodash';
+import './style.css';
+import TaskObject from '../modules/taskObj';
+import AddDelete from '../modules/tasks';
+import Store from '../modules/store';
+import StatusUpdate from '../modules/statusUpdate';
+
```
Comment (Talha-Hanif5666) [inline src/index.js:7]: I suggest you to not leave any unnecessarily blank lines in your code. This makes your code unclean and complex. Kindly fix them and remove the unnecessarily blank lines from your code.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+function component() {
+    document.addEventListener('DOMContentLoaded', AddDelete.displayTasks());
+
```
Comment (Talha-Hanif5666) [inline src/index.js]: I suggest you to not leave any unnecessarily blank lines in your code. This makes your code unclean and complex. Kindly fix them and remove the unnecessarily blank lines from your code.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    document.querySelector('.book-input').addEventListener('keyup', (event) => {
+      event.preventDefault();
+      const enterText = document.querySelector('.book-input').value;
+      if(event.keyCode === 13 && enterText !== '') {
+        const description = document.querySelector('.book-input').value;
+        const id = Store.getLocalStorage().length + 1;
+        const completed = false;
+
```
Comment (Talha-Hanif5666) [inline src/index.js]: I suggest you to not leave any unnecessarily blank lines in your code. This makes your code unclean and complex. Kindly fix them and remove the unnecessarily blank lines from your code.

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    const updateComplete = StatusUpdate.updateAllCheckbox();
+    updateComplete.forEach((box, index) => {
+      box.addEventListener('change', () => {
+        StatusUpdate.completedCheckbox();
+        console.log(updateComplete);
+      })
+    })
+    
+
```
Comment (Talha-Hanif5666) [inline src/index.js]: Kindly remove the extra blank lines.

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+.clear-button {
+  width: 100%;
+  padding: 25px 0;
+  border: none;
+  cursor: pointer;
+  font-family: 'Lato', sans-serif;
+  font-size: 18px;
+  color: #272a31;
+}
```
Comment (Talha-Hanif5666) [inline src/style.css:87]: I suggest you kindly add an empty line at the end of the CSS file.

**Review by Talha-Hanif5666 — CHANGES_REQUESTED** [review]
Hi @Fene-87,
Your project page is looking fantastic so far 💯  !
However, there is still some room for improvement, which can help you have better code and make your project professional ♻️ 

157397398-00904e26-60ec-46cd-9510-240661c250ef

### **Highlights** 🔆 
- Great work using the right GitHub Flow 💯 
- Linters are passing correctly ✔️ 
- Good commit history 👍 
Congrats on your work so far 👏 !

### **Required Changes**
- There are some issues that you still need to work on to go to the next project, but you are almost there!
- For more information, check the comments under the review 🔢

File: src/style.css (hunk 5 above)
Comment (Fene-87) [reply src/style.css:87]: Thanks for the review. I will implement the changes ASAP.😃

**Review by Talha-Hanif5666 — APPROVED** [review]
### **Hi @Fene-87 !**
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 👍 
### **Highlights**
- All linters passing ✔️ 
- All requirements completed ✔️ 
- Professional readme file ✔️ 

68747470733a2f2f6d65646961342e67697068792e636f6d2f6d656469612f436c63577241526b727131474d2f67697068792e676966

Cheers and Happy coding! 👏 👏 👏 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@Talha-Hanif5666 ] in your question so I can receive the notification.

**Comment by Fene-87** [comment]
Okay, thank you for your feedback

**Pull request merged**